### DISCUSSION TITLE
Is the PostgreSQL Protocol supported on the cloud version

### DISCUSSION THREAD

**Discussion by sipsorcery** [ravendb/ravendb#17042] [Q&A] [answered]
Can I connect to a cloud version or RavenDB using https://ravendb.net/docs/article-page/5.4/csharp/integrations/postgresql-protocol/overview?

**Comment by ayende** [comment, marked as answer]
Yes, you need to create a support call to enable that, and it is only available on certain editions.
Create a support call on the portal to get the full details relevant to your account.
